controllers/storagecluster/cephblockpools.go

```diff
@@ -15,19 +15,50 @@ import (
 
 type ocsCephBlockPools struct{}
 
+// ensures that peer cluster secret exists and adds it to CephBlockPool
+func (r *StorageClusterReconciler) addPeerSecretsToCephBlockPool(initData *ocsv1.StorageCluster) (*cephv1.MirroringPeerSpec, error) {
+	secretNames := []string{}
+
+	if len(initData.Spec.Mirroring.PeerSecretNames) == 0 {
+		return nil, fmt.Errorf("Mirroring is enabled but SecretNames are not provided")
+	}
+	for _, secretName := range initData.Spec.Mirroring.PeerSecretNames {
+		_, err := r.retrieveSecret(secretName, initData)
+		if err != nil {
+			return nil, fmt.Errorf("Peer cluster token could not be retrieved")
```

Comment:
```suggestion
			return nil, fmt.Errorf("peer cluster token could not be retrieved")
```

Reply:
Please include the secret name as well in the log message

Reply:
ack